UnityProject/Assets/Prefabs/Items/Medical/Devices/DefibrillatorPaddlesNT.prefab

```diff
@@ -136,7 +136,7 @@ PrefabInstance:
     - target: {fileID: 9019548994788298775, guid: 89c3f4c525ddf014d8dbe780d74999b1,
         type: 3}
       propertyPath: soundReady.AssetAddress
-      value: Assets/Prefabs/Items/defib/DefibReady.prefab
+      value: null
```

Comment:
what are these null values?

Reply:
No clue.

Reply:
I think the sounds just might have been removed for some reason?

Reply:
That does not appear to be a real path though.

Reply:
can you check if it works as intended?

Reply:
If the sounds are null then it won't work as intended and will just throw an error.

Reply:
I am pretty sure that absolutely no audio works now.

Reply:
or maybe i am just doing everything completely wrong

Reply:
what do you mean "no audio works now" ?

Reply:
I think I fucked up addressables by deleting a directory that was not actually something that would be reacquired.